.github/workflows/_test_t5x.yaml

```diff
@@ -101,14 +110,24 @@ jobs:
           mkdir output/
           rsync -rtz --progress \
             ${{ secrets.CLUSTER_LOGIN_USER }}@${{ vars.HOSTNAME_SLURM_LOGIN }}:${{ steps.meta.outputs.LOG_FILE }} \
-            output/${{ steps.meta.outputs.TEST_CASE_NAME }}.log
+            output/${{ steps.meta.outputs.TEST_CASE_NAME }}.log || true
```

Comment:
@yhtang There's an issue #147 with DNS being un-reliable. Any thoughts on this line? Should we exponentially backoff and retry?